scripts/bug_retriever.py

```diff
@@ -17,18 +17,32 @@ class Retriever(object):
     def retrieve_bugs(self, limit: int = None) -> None:
         bugzilla.set_token(get_secret("BUGZILLA_TOKEN"))
 
+        last_modified = None
         db.download(bugzilla.BUGS_DB)
 
         # Get IDs of bugs changed since last run.
-        last_modified = db.last_modified(bugzilla.BUGS_DB)
-        logger.info(
-            f"Retrieving IDs of bugs modified since the last run on {last_modified}"
-        )
-        changed_ids = set(
-            bugzilla.get_ids(
-                {"f1": "delta_ts", "o1": "greaterthaneq", "v1": last_modified.date()}
+        try:
+            last_modified = db.last_modified(bugzilla.BUGS_DB)
+        except Exception as e:
```

Comment:
This will need to change, as the exception thrown by `last_modified` has changed.

Reply:
Done